test_fakeredis.py

```diff
@@ -4178,6 +4178,45 @@ def test_lua_log(self):
                                          'INFO:%s:notice' % logger.name,
                                          'WARNING:%s:warning' % logger.name])
 
+    def test_lua_log_no_message(self):
+        if not isinstance(self.redis, (fakeredis.FakeRedis, fakeredis.FakeStrictRedis)):
+            pytest.skip("Works only on fakeredis, as testing real redis will require access to its logs")
```

Comment:
Is this skip necessary? Won't real redis raise the same ResponseError? Same applies to the other tests that expect a ResponseError.